### ISSUE TITLE
.Net internal package updates

### ISSUE THREAD

**Issue by ids-pfinn** [dependabot/feedback#668]
I'm pretty new to dependabot. Apologies for any confusion on my part.

## What I did
I set up the Dependabot Preview app for one of my repos and have it watching a packages.config in a sub directory. The dependency I would like dependabot to upgrade is only published to a private repository. Through the UI I setup a config variable pointing to the private repository.
The check has ran multiple times without creating a PR to update the packages.config.

## Question
1. Should dependabot be able to update packages.config from an private nuget feed?
1. Do I have access to any logs as to why this isn't working?
1. Suggested next steps?

Thanks!

**Comment by feelepxyz** [comment]
@ids-pfinn yes private nuget feeds should work if you'v configured the config variable. I can check the logs on our end, which repo are you having issues with? If it's a private repo please email it to [support@dependabot.com](mailto:support@dependabot.com).

**Comment by ids-pfinn** [comment]
Email sent. Thank you!
